dune-project

```diff
@@ -0,0 +1,3 @@
+(lang dune 1.7)
+
+(name mirage-block-solo5)
```

Comment:
there's an unneeded newline in this file.
would `dune 1.0` suffice here, or are there any 1.7 features strongly necessary?

Reply:
`dune 1.0` is probably enough :)